LetSwift/ApplicationViewControllers/Login/LoginViewController.swift

```diff
@@ -78,16 +79,9 @@ final class LoginViewController: UIViewController {
         animatedGreetingLabel.adjustsFontSizeToFitWidth = true
         loginPurposeDescriptionLabel.attributedText = loginPurposeDescriptionLabel.text?.attributed(withSpacing: 1.0)
     }
-
-    private func createPrintAttributedText(_ label: String) -> NSAttributedString {
-        return "print".attributed(withColor: .swiftOrange) +
-            "(\"".attributed(withColor: .coolGrey) +
-            label.attributed() +
-            "\")".attributed(withColor: .coolGrey)
-    }
-
+    
     private func animateLabel(randomHello: String) {
```

Comment:
I'm considering if this method should know that it takes as parameter randomHello or just text or even animateLabel(with text: String). But it's not a part of this PR.

Reply:
Sure, it should. The whole label animation logic and behavior needs some refactoring.